src/garage/tf/algos/npo.py

```diff
@@ -177,76 +177,45 @@ def train(self, trainer):
         last_return = None
 
         for _ in trainer.step_epochs():
-            trainer.step_path = trainer.obtain_samples(trainer.step_itr)
-            last_return = self._train_once(trainer.step_itr, trainer.step_path)
+            episodes = trainer.obtain_episodes(trainer.step_itr)
```

Comment:
You should still save them to `trainer.step_path`, even though they're episodes. This is needed for `save_paths` to work.